### DISCUSSION TITLE
Do newly added devices get auto-discovered?

### DISCUSSION THREAD

**Discussion by betabob** [potmat/homebridge-google-nest-sdm#177] [Q&A]
I setup the plugin awhile back and it's been working fine. However, I recently added a new Nest Cam to my home, but it never shows up as a Homebridge accessory, even after restarting the plugin. Is there some type of procedure that's required in order to add new devices?

**Comment by potmat** [comment]
From the FAQ:

<img width="851" alt="image" src="https://github.com/user-attachments/assets/3b5fcba7-ba26-418a-ac16-75af9dfcd08f">
